components/pagination/stories/pagination.stories.js

```diff
@@ -57,3 +74,15 @@ export default {
 
 export const Default = PaginationGroup.bind({});
 Default.args = {};
+
+/**
+ * Pagination's explicit variant uses the text field component to represent the current page, and text with the total number of pages. Along with action buttons for previous and next navigation.
+ */
+export const Explicit = Template.bind({});
+Explicit.tags = ["docs-only"];
```

Comment:
Can you update the tags to our new standards when you get a chance?

Reply:
Sure. This is done for all of the tags in this PR.